tools/compare-parsing.py

```diff
@@ -0,0 +1,47 @@
+# compare-parsing.py parses all programs in a CSV files with header location,code
```

Comment:
Would it be possible to migrate this logic to Go?
I'm a bit worried about using multiple languages in a single repo, as it would become harder to maintain. i.e: Need expertise in multiple languages.
(Code base is mainly Go, and there are some minor usages of Typescript etc. as well. Having another language would add one more to the technology stack)

Reply:
Good point! Ported the tool to Go in [d6bc6fa](https://github.com/onflow/cadence/pull/2136/commits/d6bc6fad287e55be5b5d9e215b2975ffc45ede6e)